website/source/docs/agent/options.html.md

```diff
@@ -1110,6 +1110,11 @@ default will automatically work with some tooling.
   `server_name`) to set up the client for HTTP or gRPC health checks. This allows services requiring 2-way TLS to
   be checked using the agent's credentials. This was added in Consul 1.0.1 and defaults to false.
 
+* <a name="enable_central_service_config"></a><a href="#enable_central_service_config">`enable_central_service_config`</a>
+  When set, the Consul agent will merge any service registrations with centrally configured defaults for that service, if 
```

Comment:
When set, the Consul agent will look for any centralized service configurations that match a registering service instance. If it finds any, the agent will merge the centralized defaults with the service instance configuration.

Reply:
Just an attempt to make this description a bit clearer. Feel free to take it or leave it